website/docs/r/nsxt_network_imported.html.markdown

```diff
@@ -64,12 +94,15 @@ and inherited from provider configuration)
 * `vdc` - (Deprecated; Optional) The name of VDC to use. **Deprecated**  in favor of new field
   `owner_id` which supports VDC and VDC Group IDs.
 * `name` - (Required) A unique name for the network
-* `nsxt_logical_switch_name` - (Required) Unique name of an existing NSX-T segment. 
+* `nsxt_logical_switch_name` - (Optional) Unique name of an existing NSX-T segment. 
```

Comment:
What about stating that the user should provide either `nsxt_logical_switch_name` OR `dvpg_name`?